websites/K/Keep Talking and Nobody Explodes/presence.ts

```diff
@@ -1,159 +1,213 @@
 const presence = new Presence({
     clientId: "681116862930747520"
   }),
-  strings = presence.getStrings({
-    browsing: "presence.activity.browsing",
-    reading: "presence.activity.reading"
-  });
+  strings = getStrings();
```

Comment:
Handle language changing & define strings in the event instead of awaiting it each time